### PR TITLE
new icon: playwright (original, plain) 

### LINKED ISSUE #1426: [ICON REQUEST]: Playwright
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Playwright enables reliable end-to-end testing for modern web apps.

### Links and sources

https://playwright.dev
https://github.com/microsoft/playwright

#### Official Logo

https://playwright.dev/img/playwright-logo.svg

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by oguzakif**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1426**

## Link to prove your SVG is correct and up-to-date.

+ <a href="https://playwright.dev/">Playwright Official Page</a>
+ <a href="https://github.com/microsoft/playwright">Github page for playwright</a>
+ Original SVG preview: 
playwright-original

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -5628,6 +5628,24 @@
         "color": "#b74af7",
         "aliases": []
     },
+    {
+        "name": "playwright",
```
Comment (Snailedlt) [inline devicon.json:5632]:
```suggestion
        "name": "playwright",
        "altnames": [],
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]

File: devicon.json (hunk 1 above)
Comment (Snailedlt) [reply devicon.json:5632]: alnames is missing

Comment (oguzakif) [reply devicon.json:5632]: Hi, I have added the altnames as you mentioned above. Thank you for your feedback.

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Looks good now!
Just something I missed in the first review, that needs to be fixed

File: icons/playwright/playwright-plain.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...-2.372-1.684 3.417-3.837 8.977-5.921 16.762-4.516 16.857-7.892 52.429 20.027 59.914 27.912 7.479 42.772-25.017 47.289-41.875 2.084-7.783 2.998-13.676 3.25-17.476.287-4.305-2.67-3.055-8.324-2.064ZM53.28 55.282s4.4-6.843 11.862-4.722c7.467 2.121 8.045 10.376 8.045 10.376L53.28 55.282Zm18.215 30.706c-13.125-3.845-15.15-14.311-15.15-14.311l35.259 9.858c0-.002-7.117 8.25-20.109 4.453Zm12.466-21.51s4.394-6.838 11.854-4.711c7.46 2.124 8.048 10.379 8.048 10.379l-19.902-5.668Z"/><path d="M44.762 78.733 31.787 82.41s1.41-8.029 10.968-11.212l-7.347-27.573-.635.193c-9.111 2.457-16.476 1.805-19.55 1.273-4.357-.751-6.636-1.708-6.422 1.606.186 2.923.882 7.454 2.477 13.44 3.45 12.961 14.854 37.937 36.405 32.132l.635-.199-3.555-13.337ZM19.548 60.315l15.316-4.035s-.446 5.892-6.188 7.405c-5.743 1.512-9.128-3.371-9.128-3.371Z"/><path d="m72.086 86.132-.594-.144c-13.125-3.844-15.15-14.311-15.15-14.311l18.182 5.082L84.15 39.77l-.116-.031c-11.807-3.162-19.64-8.692-22.744-11.292-4.4-3.685-6.335-6.246-8.24-2.372-1.682 3.417-3.836 8.977-5.92 16.762-4.516 16.857-7.892 52.429 20.027 59.914l.572.129 4.357-16.748Zm-18.807-30.85s4.4-6.843 11.862-4.722c7.467 2.121 8.045 10.376 8.045 10.376l-19.907-5.654Z"/></svg>
```
[the remark this answers is not in the document]
Comment (oguzakif) [reply icons/playwright/playwright-plain.svg]: Hi, I have combined multiple paths into single path as you mentioned. Thank you for your feedback.

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy_check_mark: 

Thanks for a fast and clean contribution! :100:

**Review by Panquesito7 — APPROVED** [review]
Great work! Thank you. 🚀

**Comment by Panquesito7** [comment]
One question before merging: should we add wordmark versions?

**Comment by oguzakif** [comment]
> One question before merging: should we add wordmark versions?

I did not add it because there is no wordmark version of the logo anywhere on Playwright's site. If its needed, I can add them too.

**Pull request merged**